src/v/storage/segment.cc

```diff
@@ -336,14 +337,17 @@ ss::future<> segment::do_flush() {
     if (!_appender) {
         return ss::make_ready_future<>();
     }
-    auto o = _tracker.dirty_offset;
+    auto o = _tracker.get_dirty_offset();
```

Comment:
it's nice to separate out the simple mechanical changes like renames (`get_xxx_offset`) into a separate commit so that the commits with more complicated changes can be focused.